src/models/index.js

```diff
@@ -4,3 +4,5 @@ export { ShippingContact } from './ShippingContact'
 export { BillingContact } from './BillingContact'
 export { BillingPreferences } from './BillingPreferences'
 export { Account } from './Account'
+export { InstrumentModel } from './InstrumentModel'
```

Comment:
The convention was to drop Model, but I can see that it has its use and why you went with it. We need to pick one convention for naming, but don't have Model appended to any other class in this library.